home/tests/test_assessment_import_export.py

```diff
@@ -0,0 +1,512 @@
+
+        (This uses bulk_assessments.csv.)
+
+        """
+
+        home_page = HomePage.objects.first()
+        main_menu = PageBuilder.build_cpi(home_page, "main-menu", "Main Menu")
+        PageBuilder.build_cp(
+            parent=main_menu,
+            slug="high-inflection",
+            title="High Inflection",
+            bodies=[
+                WABody("High Inflection", [WABlk("*High Inflection Page")]),
+                MBody("High inflection", [MBlk("High Inflection Page")]),
+            ],
+        )
+        PageBuilder.build_cp(
+            parent=main_menu,
+            slug="medium-score",
+            title="Medium Score",
+            bodies=[
+                WABody("Medium Score", [WABlk("*Medium Inflection Page")]),
+                MBody("Medium Score", [MBlk("Medium Inflection Page")]),
+            ],
+        )
+
+        PageBuilder.build_cp(
+            parent=main_menu,
+            slug="low-score",
+            title="Low Score",
+            bodies=[
+                WABody("Low Score", [WABlk("*Low Inflection Page")]),
+                MBody("Low Score", [MBlk("Low Inflection Page")]),
+            ],
+        )
+
+        csv_bytes = csv_impexp.import_file("bulk_assessments.csv")
+        content = csv_impexp.export_assessment()
+        src, dst = csv_impexp.csvs2dicts(csv_bytes, content)
+        assert dst == src
+
+    def test_comma_separated_answers(self, csv_impexp: ImportExport) -> None:
+        """
+        Importing a CSV file with multiple assessments. Some with comma
+        separated answers and some without commas.
+
+        (This uses comma_sep_assessments.csv.)
+
+        """
+
+        home_page = HomePage.objects.first()
+        main_menu = PageBuilder.build_cpi(home_page, "main-menu", "Main Menu")
+        PageBuilder.build_cp(
+            parent=main_menu,
+            slug="high-inflection",
+            title="High Inflection",
+            bodies=[
+                WABody("High Inflection", [WABlk("*High Inflection Page")]),
+                MBody("High inflection", [MBlk("High Inflection Page")]),
+            ],
+        )
+        PageBuilder.build_cp(
+            parent=main_menu,
+            slug="medium-score",
+            title="Medium Score",
+            bodies=[
+                WABody("Medium Score", [WABlk("*Medium Inflection Page")]),
+                MBody("Medium Score", [MBlk("Medium Inflection Page")]),
+            ],
+        )
+
+        PageBuilder.build_cp(
+            parent=main_menu,
+            slug="low-score",
+            title="Low Score",
+            bodies=[
+                WABody("Low Score", [WABlk("*Low Inflection Page")]),
+                MBody("Low Score", [MBlk("Low Inflection Page")]),
+            ],
+        )
+
+        csv_bytes = csv_impexp.import_file("comma_separated_answers.csv")
+        content = csv_impexp.export_assessment()
+        src, dst = csv_impexp.csvs2dicts(csv_bytes, content)
+        assert dst == src
```

Comment:
I think we can add another test here to deal with newlines in the question, I don't see that being tested anywhere.